sections/alert_production_pipelines.tex

```diff
@@ -0,0 +1,510 @@
+\label{sec:apSingleFrameProcessing}
+
+The Single Frame Processing (SFM) Pipeline (see Figure~\ref{fig:apSFM}) is responsible for reducing raw or camera-corrected image data to \emph{calibrated exposures}, the detection and measurement of \Sources (using the components functionally  part of the Object Characterization Pipeline), the characterization of the point-spread-function (PSF), and the generation of an astrometric solution for an image. Calibrated exposures produced by the SFM pipeline must possess all information necessary for measurement of source properties by single-epoch Object Characterization algorithms.
+
+Astrometric and photometric calibration requires the detection and measurement of the properties of \Sources on a CCD. Accurate centroids and fluxes for these \Sources require an estimation of the PSF and background, which in turn requires knowledge of the positions of the \Sources on an image. The SFM pipeline will, therefore, iterate over background estimation (see \ref{sec:apPSFBackground}) and source measurement (see \ref{sec:apSourcemeasurement})
+
+The SFM pipeline will be implemented as a flexible framework where new processing steps can be added without modifying the stack code (this would include the ability to process non-crosstalk corrected images should a network outage between the base and processing center result in  only the raw data being available). The pipeline, or a subset of the pipeline, should be capable of being run at the telescope facility during commissioning and operations.
+
+\begin{figure}[th]
+\begin{center}
+\includegraphics[width=0.9\textwidth]{figures/SFM.png}
+\caption{\label{fig:apSFM} Single frame processing of the nightly data: instrument signature removal, astrometric and photometric calibration, background and PSF estimation from the cross-talk corrected camera images.}
+\end{center}
+\end{figure}
+
+%SFM pipeline functions include:
+%\begin{itemize}
+%\item Assembly of per-amplifier images to an image of the entire CCD;
+%\item Instrumental Signature Removal;
+%\item Cosmic ray rejection and snap combining;
+%\item Per-CCD determination of zeropoint and aperture corrections;
+%\item Per-CCD PSF determination;
+%\item Per-CCD WCS determination and astrometric registration of images;
+%\item Per-CCD sky background determination;
+%\item Source detection and measurement on single frame images
+%\item Generation of metadata required by the OCS
+%\end{itemize}
+
+\subsubsection{Input Data}
+\label{sec:apSFMinput}
+
+\paragraph*{Raw Camera Images:} Amplifier images that have been corrected for crosstalk and bias by the camera software. All images from a visit should be available to the task (including snaps). An approximate WCS is assumed to be available as metadata derived from the Telescope Control System with an absolute pointing uncertainty (for a full focal plane) of 2 arcseconds (OSS-REQ-0298) and the field rotation known to an accuracy of 32 arcseconds (LTS-206).
```

Comment:
Where does the 32 arcseconds come from in LTS-206? LTS-206 is a bit problematic because it doesn't have requirements numbers and parameters like this aren't explicitly called out. I see section 3.4.3 say that the rotator encoder accuracy is 0.002 degrees (stated as one pixel at the edge of the focal plane). It's somewhat confusing that the number in LDM-151 can not be found directly in LTS-206.